README.md

```diff
@@ -53,6 +53,8 @@ $ npm i -g aws-cdk
 Initialize a project:
 
 ```bash
+$ mkdir ~/hello-cdk
```

Comment:
I wouldn’t be opinionated about where to create the project directory: `mkdir hello-cdk`

Reply:
Good point